pbs/pbsresponse.go

```diff
@@ -1,26 +1,38 @@
 package pbs
 
+// PBSBid is a bid from the auction. These are produced by Adapters, and target a particular Ad Unit.
 type PBSBid struct {
-	BidID             string            `json:"bid_id"`
-	AdUnitCode        string            `json:"code"`
-	Creative_id       string            `json:"creative_id,omitempty"`
-	BidderCode        string            `json:"bidder"`
-	BidHash           string            `json:"-"` // this is the hash of the bidder's unique bid identifier for blockchain. Should not be sent to browser.
-	Price             float64           `json:"price"`
-	Currency          string            `json:"currency,omitempty"`
-	NURL              string            `json:"nurl,omitempty"`
-	Adm               string            `json:"adm,omitempty"`
-	Width             uint64            `json:"width,omitempty"`
+	// BidID identifies the Bid Request within the Ad Unit which this Bid targets. It should match one of
+	// the values inside PBSRequest.AdUnits[i].Bids[j].BidID.
+	BidID string `json:"bid_id"`
+	// AdUnitCode identifies the AdUnit which this Bid targets.
+	// It should match one of PBSRequest.AdUnits[i].Code, where "i" matches the AdUnit used in
+	// as BidID.
+	AdUnitCode  string `json:"code"`
+	Creative_id string `json:"creative_id,omitempty"`
```

Comment:
The unique ID of the creative from the adapter. Useful for ad quality / malware analysis, for instance (so you can call and say "your creative ABCD is malware!!") This is a passthrough field for PBS